vetalapo/eCommerce
# Tests

1. Task: tests for Login page
2. Screenshot: none
3. Change description: renders the login form, displays an error message when login fails, calls the login method on form submit
4. Change rationale: ensure the reliability and robustness of the implemented features
